### DISCUSSION TITLE
Index/import on one machine, transfer to another?

### DISCUSSION THREAD

**Discussion by Denperidge** [photoprism/photoprism#1304] [Q&A] [answered]
Hello there!

I've got a Raspberry Pi 3 that I'd like to use for Photoprism. However, my image library exceeds 40GB, and I do not want to think about how long indexing or importing would take on that on initial setup.
Is it possible to run Photoprism on one machine, let it index and/or import everything, and then send over the Storage+Pictures to the RPi?
(Side question: should you store your Storage on a SSD, your pictures, or both?)

Thanks in advance!

**Comment by alexisLefebvre** [comment, marked as answer]
See https://github.com/photoprism/photoprism/issues/1100#issuecomment-786771690 and #1145

**Reply by Denperidge** [reply]
My thanks for the links!

**Comment by lastzero** [comment]
It can be done in theory. 40 GB doesn't seem a lot though. 500 GB are common among our users. Indexing might take a bit longer compared to a strong server, but setting this all up properly also takes time and you don't need to do this repeatedly.

**Reply by Denperidge** [reply]
That's fair, thankyou! How long should importing/indexing take? I let it run overnight on my Windows machine (7th generation quad core CPU) but it was still not done the evening after. Although I admittedly did let it run on an external hard drive, which might be the reason for that?

**Reply by lastzero** [reply]
Yes, as mentioned in our docs, fast I/O - ideally a local SSD, not connected via slow USB - is very helpful when indexing sometimes very large photo & video files.
